bundle/Resources/config/services/items.yaml

```diff
@@ -17,3 +17,15 @@ services:
             - "@router"
         tags:
             - { name: netgen_layouts.cms_value_url_generator, value_type: sylius_product }
+
+    netgen_layouts.sylius.item.value_converter.taxon:
```

Comment:
Do taxons have URLs in Sylius? If so, then you're missing the URL generator class.

Reply:
@emodric @igorkrz They do: https://demo.sylius.com/en_US/taxons/fashion-category/dresses

That's basically the purpose if this PR, so that we can eg. list all taxons as tiles with link to their page (eg. categories).

Reply:
Added in https://github.com/netgen-layouts/layouts-sylius/pull/19/commits/98dafbcd4286f9b181013dccc0879bac31ec1ac4